docker/test/stateful/Dockerfile

```diff
@@ -6,23 +6,14 @@ FROM clickhouse/stateless-test:$FROM_TAG
 RUN apt-get update -y \
     && env DEBIAN_FRONTEND=noninteractive \
         apt-get install --yes --no-install-recommends \
+        llvm-9 \
         python3-requests \
-        llvm-9
+    && apt-get clean
 
 COPY s3downloader /s3downloader
 
 ENV S3_URL="https://clickhouse-datasets.s3.amazonaws.com"
 ENV DATASETS="hits visits"
-ENV EXPORT_S3_STORAGE_POLICIES=1
-
-# Download Minio-related binaries
-RUN arch=${TARGETARCH:-amd64} \
-    && if [ "$arch" = "amd64" ] ; then wget "https://dl.min.io/server/minio/release/linux-${arch}/archive/minio-20220103182258.0.0.x86_64.rpm"; else wget "https://dl.min.io/server/minio/release/linux-${arch}/archive/minio-20220103182258.0.0.aarch64.rpm" ; fi \
-    && wget "https://dl.min.io/client/mc/release/linux-${arch}/mc" \
-    && chmod +x ./mc
-ENV MINIO_ROOT_USER="clickhouse"
-ENV MINIO_ROOT_PASSWORD="clickhouse"
-COPY setup_minio.sh /
```

Comment:
I think it's better to download it when building the image and then just use this image (we have our own cache for images, right?). Otherwise minio's repo may ban our IPs if we will download it for each tests run.

Reply:
The image is built `FROM clickhouse/stateless-test:$FROM_TAG`. It's downloaded there already.

Reply:
Then let's remove downloading from `setup_minio.sh`, because it's already donloaded